docs/cli-reference/dfx-completion.mdx

```diff
@@ -0,0 +1,52 @@
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# dfx completion
+
+<MarkdownChipRow labels={["Reference"]} />
+
+Generate a shell completion script for `dfx`.
+
+## Basic usage
+
+``` bash
+dfx completion [option] SHELL
+```
+
+The command takes installed extensions into account when generating the
+completion script, so you should regenerate it after installing or removing
+extensions.
+
+The values supported for SHELL are `bash`, `elvish`, `fish`, `powershell`,
+and `zsh`. The default value is `bash`.
+
+## Options
+
+You can use the following option with the `dfx completion` command.
+
+| Option                | Description                        -|
+|-----------------------|-------------------------------------|
+| `--bin-name <name>`   | Name of the binary.  Default `dfx`. |
```

Comment:
This option made me confused.

"Can `dfx completion` generate completion for some other binaries other than `dfx`?"

After checking the docs of `clap_complete::generate()`, I finally realized that this option is for the case that a user may install the `dfx` with a different name.

I'm wondering if we can improve the explanation. Or what about rename the option to something like `--rename-dfx`?